library/Icinga/Web/RememberMeUserDevicesList.php

```diff
@@ -0,0 +1,144 @@
+<?php
+/* Icinga Web 2 | (c) 2020 Icinga GmbH | GPLv2+ */
+namespace Icinga\Web;
+
+use ipl\Html\BaseHtmlElement;
+use ipl\Html\Html;
+use ipl\Web\Url;
+use ipl\Web\Widget\Icon;
+use ipl\Web\Widget\Link;
+
+class RememberMeUserDevicesList extends BaseHtmlElement
+{
+    protected $tag = 'table';
+
+    protected $defaultAttributes = [
+        'class'             => 'common-table',
+        'data-base-target'  => '_self',
+        'title'             => 'click to remove this cookie from the database'
+    ];
+
+    /**
+     * @var array
+     */
+    protected $devicesList;
+
+    /**
+     * @var string
+     */
+    protected $username;
+
+    /**
+     * @var string
+     */
+    protected $url;
+
+    /**
+     * @return string
+     */
+    public function getUrl()
+    {
+        return $this->url;
+    }
+
+    /**
+     * @param string $url
+     *
+     * @return $this
+     */
+    public function setUrl($url)
+    {
+        $this->url = $url;
+
+        return $this;
+    }
+
+    /**
+     * @return string
+     */
+    public function getUsername()
+    {
+        return $this->username;
+    }
+
+    /**
+     * @param string $username
+     *
+     * @return $this
+     */
+    public function setUsername($username)
+    {
+        $this->username = $username;
+
+        return $this;
+    }
+
+    /**
+     * @return array List of devices. Each device contains user agent and fingerprint string
+     */
+    public function getDevicesList()
+    {
+        return $this->devicesList;
+    }
+
+    /**
+     * @param $devicesList
+     *
+     * @return $this
+     */
+    public function setDevicesList($devicesList)
+    {
+        $this->devicesList = $devicesList;
+
+        return $this;
+    }
+
+    protected function assemble()
+    {
+        $thead = Html::tag('thead');
+        $theadRow = Html::tag('tr')
+            ->add(Html::tag(
+                'th',
+                t("List of devices and browsers {$this->getUsername()} is currently logged in:")
```

Comment:
Use sprintf to inject the username here. The `{ $expr }` syntax is invalid if the string is localizable.